Assets/Scripts/Menu.cs

```diff
@@ -11,57 +14,110 @@ public class Menu : MonoBehaviour
     private Button tutorialButton;
     private Button quitButton;
     private Label versionNumber;
+
+    private Dictionary<string, VisualElement> sceneRootElements = new Dictionary<string, VisualElement>();
+
     void Awake()
     {
         VisualElement document = GetComponent<UIDocument>().rootVisualElement;
-        playButton = document.Q<Button>("PlayButton");
+        playButton = document.Q<Button>("Play");
         playButton.RegisterCallback<ClickEvent>(Play);
 
-        highscoreButton = document.Q("HighscoreButton") as Button;
-        highscoreButton.RegisterCallback<ClickEvent>(Highscore);
-
-        tutorialButton = document.Q("TutorialButton") as Button;
-        tutorialButton.RegisterCallback<ClickEvent>(Tutorial);
+        highscoreButton = document.Q<Button>("Highscores");
+        tutorialButton = document.Q<Button>("Tutorial");
 
-        quitButton = document.Q("QuitButton") as Button;
+        quitButton = document.Q<Button>("Quit");
         quitButton.RegisterCallback<ClickEvent>(Quit);
 
         versionNumber = document.Q<Label>("VersionNumber");
         versionNumber.text = "Alpha V0.9.0";
 
     }
+
+    private void Start()
+    {
+        Application.targetFrameRate = 60;
+        SFXManager.Instance.PlayBackgroundMusic(SFXManager.Instance.TitleScreen);
+        StartCoroutine(LoadBackgroundScene("Highscores", highscoreButton));
+        StartCoroutine(LoadBackgroundScene("Tutorial", tutorialButton));
+    }
+
     private void OnDisable()
```

Comment:
OnDisable was a function I added that I forgot to remove as unregerstering callbacks isn't really needed

Reply:
Resolved